daemon/seccomp_linux_test.go

```diff
@@ -25,6 +25,8 @@ func TestWithSeccomp(t *testing.T) {
 		comment string
 	}
 
+	defaultErrnoRet := uint(1)
```

Comment:
can we comment that this is EPERM or use a constant?

Reply:
Sure, adding a comment!

Reply:
should we use syscall.EPERM (or golang.org/x/sys/unix.EPERM) ?

Reply:
Looks like https://github.com/moby/moby/pull/42681 used unix.NOSYS, so probably makes sense to do that

Reply:
Brain fart on my end, this file doesn't need any change at all! All tests make sense without any change, and the test that loads the default profile compares against what `seccomp.GetDefaultProfile()` gives, so it works just fine!

Removing all the changes to `daemon/` files. Sorry! :)

Reply:
Marking it as resolved as this file is not changed anymore :D